storage/service/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+mod reader_threads;
```

Comment:
Please copy `cuprate_blockchain`'s `README.md' and adjust for this crate.

```suggestion
#![doc = include_str!("../README.md")]

mod reader_threads;
```

Reply:
There isn't really a lot I can copy from `cuprate_blockchain` so I just created a new file from scratch